app/src/renderer/components/wallet/PageWallet.vue

```diff
@@ -25,7 +25,7 @@
       @show-modal="showModal"
     />
     <tm-btn
-      v-if="enableFaucet"
+      v-if="session.signedIn && enableFaucet"
```

Comment:
I still have my stomach mur when I see hiding content this way.

Reply:
i appreciate you saying that every time i hide a button 😆

i agree with you but this feature wasn't implemented properly then. what should happen if someone clicks the faucet button when they are not signed in? nothing even happens when the user _is_ signed in 😕

Reply:
alright

Reply:
gneh, I would put this condition on line 61, so we also test it

Reply:
done.